syntax = "proto3";
package mars.stn;


option java_package = "win.liyufan.im.proto";
option java_outer_classname = "MessageContentOuterClass";

//option objc_class_prefix = "LYF";


enum ContentType {
    Text = 0;
    Image = 1;
    Voice = 2;
    Location = 3;
    Video = 4;
    RichMedia = 5;
    Custom = 6;
}

message MessageContent {
  ContentType type = 1;
  string searchable_content = 2;
  string push_content = 3;
  string content = 4;
  bytes data = 5;
  int32 mediaType = 6;
  string remoteMediaUrl = 7;
}
